app/src/main/java/com/babylon/wallet/android/domain/model/assets/AccountWithAssets.kt

```diff
@@ -41,30 +42,28 @@ data class Assets(
         }
     } == true
 
+    // todo remove
+    fun allSize() = poolUnitsSize() + fungibles.size + nonFungibles.size
```

Comment:
TODO

Reply:
You are right forgot about that will remove.